validator/client/beacon-api/status.go

```diff
@@ -73,8 +75,23 @@ func (c *beaconApiValidatorClient) getValidatorsStatusResponse(ctx context.Conte
 		return nil, nil, nil, errors.Wrap(err, "failed to get state validators")
 	}
 
-	isLastActivatedValidatorIndexRetrieved := false
-	var lastActivatedValidatorIndex uint64 = 0
+	// Get all validator count.
+	validatorsCountResponse, err := c.validatorCountProvider.GetValidatorCount(ctx, "head", nil)
+	if err != nil && !errors.Is(err, iface.ErrNotSupported) {
+		return nil, nil, nil, errors.Wrap(err, "failed to get total validator count")
+	}
+
+	var total, pending uint64
+	for _, valCount := range validatorsCountResponse {
+		if valCount.Status == "pending" {
```

Comment:
This endpoint can return anything, `pending`, `pending_initialized`, `pending_queued`, etc. Since the latter two are sub-statuses of pending that is why I just went with status `pending`.